html/popup.html

```diff
@@ -15,7 +15,7 @@
         <p id="start-stream"></p>
     </div>
     <div>
-        <i>Tips: You can hide the player with Alt+Maj+T</i>
+        <i>Tips: You can hide the player with Ctrl+Shift+Space</i>
```

Comment:
From https://developer.chrome.com/apps/commands : `Please note that on Mac 'Ctrl' is automatically converted to 'Command'. If you want 'Ctrl' instead, please specify 'MacCtrl' under "mac". Specifying 'MacCtrl' under "default" will cause the extension to be uninstallable.`

We can't use this as CMD + Space triggers the spotlight search on macOs :)

Reply:
> * Please note that on Mac 'Ctrl' is automatically converted to 'Command'. If you want 'Ctrl' instead, please specify 'MacCtrl' under "mac". Specifying 'MacCtrl' under "default" will cause the extension to be uninstallable.

We could use ctrl + Shift + space for default and MaCtrl + Shift + Space for mac ?
What bothers me is that there is no easy way to remember this shortcut :(

Reply:
@ImFlog you're right, let's discuss about it